project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementNGSValidator.java

```diff
@@ -83,65 +82,38 @@ public static Collection<String> properties() {
     return Arrays.stream(NGS_PROPERTY.values()).map(NGS_PROPERTY::label).toList();
   }
 
-  @Override
   @PreAuthorize("hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'WRITE')")
-  public ValidationResult validate(NGSMeasurementMetadata measurementMetadata,
-      ProjectId projectId) {
+  public ValidationResult validateRegistration(MeasurementRegistrationInformationNGS metadata,
+      String experimentId, ProjectId projectId) {
     var validationPolicy = new ValidationPolicy();
-    //We want to fail early so we check first if all the mandatory fields were filled
-    ValidationResult mandatoryValidationResult = validationPolicy.validateMandatoryDataProvided(
-        measurementMetadata);
-    if (mandatoryValidationResult.containsFailures()) {
-      return mandatoryValidationResult;
+    var result = ValidationResult.successful();
+    for (String sampleId : metadata.measuredSamples()) {
+      result = result.combine(validationPolicy.validationProjectRelation(sampleId, projectId))
+          .combine(
+              validationPolicy.validationExperimentRelation(sampleId, experimentId, projectId));
     }
-    //If all fields were filled then we can validate the entries individually
-    return validationPolicy.validateSampleIds(measurementMetadata.sampleCodes())
-        .combine(validationPolicy.validateMandatoryDataProvided(measurementMetadata))
-        .combine(validationPolicy.validateOrganisation(measurementMetadata.organisationId())
-            .combine(validationPolicy.validateInstrument(measurementMetadata.instrumentCURI())));
-  }
-
-  @PreAuthorize("hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'WRITE')")
-  public ValidationResult validateRegistration(MeasurementRegistrationInformationNGS registration,
-      ProjectId projectId) {
-    var validationPolicy = new ValidationPolicy();
 
     ValidationResult mandatoryValidationResult = validationPolicy.validateMandatoryDataRegistration(
-        registration);
+        metadata);
     if (mandatoryValidationResult.containsFailures()) {
       return mandatoryValidationResult;
     }
-    return validationPolicy.validateSampleIdsAsString(registration.measuredSamples())
-        .combine(validationPolicy.validateMandatoryDataRegistration(registration))
-        .combine(validationPolicy.validateOrganisation(registration.organisationId()))
-        .combine(validationPolicy.validateInstrument(registration.instrumentCURIE()));
+    return result.combine(validationPolicy.validateSampleIdsAsString(metadata.measuredSamples()))
+        .combine(validationPolicy.validateMandatoryDataRegistration(metadata))
```

Comment:
we have the validate mandatory data policy twice now in this method. I suggest to decide for one. I have no strong opinions on the order. Maybe it makes sense to not return before all checks have been done (except the project / experiment relation)